### ISSUE TITLE
The stroke of a shape is not affected by clipping with another shape

### ISSUE THREAD

**Issue by JcToon** [2d-inc/support#198]
When you click with two shapes and one of them you add stroke, the stroke is not affected by the cut. For ways that you use the stroke, such as mouths, it is good that the stroke is cut when you use the clipping tool.
![clipping without stroke](https://user-images.githubusercontent.com/45951634/50665170-e65c1a80-0faf-11e9-952c-355552d7b76e.jpg)

**Comment by neurowave** [comment]
Yeah, this currently happens because the draw order of the teeth is above the draw order of the stroke. You can mitigate it by making the stroke draw order higher than the teeth, but you can't have a fill on the shape with the stroke.

@luigi-rosso can we inset the clipping amount by the stroke width in this case?

**Comment by JcToon** [comment]
Yes, right now what I have is another form above only with the stroke. 1 Form only with stroke (lips), 2 teeth, 3 mouth. The mouth is the one that cuts the teeth and the tongue.

**Comment by neurowave** [comment]
If you already have the stroke as a separate shape, then you can move the draw order above the teeth… that should fix it.

**Comment by JcToon** [comment]
Yes, it is just as I have it
![new file 9 - google chrome 03_01_2019 20_48_14_2](https://user-images.githubusercontent.com/45951634/50665940-e7427b80-0fb2-11e9-97e3-5b1451ff9e8e.gif)
